.vscode/settings.json

```diff
@@ -1,3 +1,28 @@
 {
-	"peacock.color": "#0b5017"
```

Comment:
This is a file that should be put on the git ignore list as it is personal to each developer :)